docs/slack.rst

```diff
@@ -275,13 +275,47 @@ In addition to the guidance that applies to all participants, we expect self-pro
   If you post about things that aren't relevant to the documentation field or the channel you're posting in, moderators might delete your messages.
   Make sure conversations stay relevant and not sales-focused, so you can have the chance to talk about work you're excited about.
 
+Common Self-Promotion Situations
+++++++++++++++++++++++++++++++++
+
+Here are some examples of common self-promotion situations and how to self-promote responsibly (and irresponsibly):
+
+- You've written a blog post (Medium, Hashnode, etc.) that you want to share with the community, but it's not closely related to an ongoing discussion.
+
+  **Do** share it in **community-showcase** and tell us why it is relevant to the practice of writing, creating, maintaining, or building docs.
```

Comment:
consistency nit

(I mean, I guess you could go the other way and change all the channel mentions to no `#`, but I think it's not obviously a channel otherwise)

```suggestion
  **Do** share it in **#community-showcase** and tell us why it is relevant to the practice of writing, creating, maintaining, or building docs.
```